packages/ui/src/components/commit-copy-actions.tsx

```diff
@@ -20,7 +35,41 @@ export const CommitCopyActions = ({ sha }: { sha: string }) => {
   return (
     <ShaBadge.Root>
       <ShaBadge.Content className="hover:bg-transparent">
-        <span className="text-13 text-foreground-3">{sha.substring(0, 7)}</span>
+        <span
+          role="button"
+          tabIndex={0}
+          onClick={() => {
+            let path
+            if (commitPath) {
+              path = `${commitPath}/${sha}`
+            } else if (inCompare || inDetails) {
+              path = `../../commits/${sha}`
```

Comment:
we are trying to avoid relative paths, maybe we can create a route for this as well?

Reply:
fixed